rgw/v2/tests/s3cmd/test_s3cmd.py

```diff
@@ -61,6 +61,56 @@ def test_exec(config, ssh_con):
 
     ip_and_port = s3cmd_reusable.get_rgw_ip_and_port(ssh_con)
 
+    if config.test_ops.get("s3cmd_get_objects", False):
+        log.info(
+            f"CEPH-83575477 - Verify s3cmd get: Bug 2174863 - [cee/sd][RGW] 's3cmd get' fails with EOF error for few objects"
+        )
+        user_info = resource_op.create_users(no_of_users_to_create=1)
+        s3_auth.do_auth(user_info[0], ip_and_port)
+        auth = Auth(user_info[0], ssh_con, ssl=config.ssl)
+        rgw_conn = auth.do_auth()
+        for bc in range(config.bucket_count):
+            bucket_name = utils.gen_bucket_name_from_userid(
+                user_info[0]["user_id"], rand_no=bc
+            )
+            s3cmd_reusable.create_bucket(bucket_name)
+            log.info(f"Bucket {bucket_name} created")
+            s3cmd_path = "/home/cephuser/venv/bin/s3cmd"
+            object_count = config.objects_count // 2
+
+            log.info(f"uploading some large objects to bucket {bucket_name}")
+            utils.exec_shell_cmd(f"fallocate -l 20m obj20m")
```

Comment:
@anrao19 , can we get these object sizes from the config?

Reply:
@hmaheswa , we do not have provision to provide size from config for s3md, we can go a head with this method for now.
we can take this as enhancement